core/admin/feedback.php

```diff
@@ -182,18 +181,6 @@ public function ajax_elementor_deactivate_feedback() {
 		wp_send_json_success();
 	}
 
-	/**
-	 * @since 2.3.0
-	 * @access protected
-	 */
-	protected function get_init_settings() {
```

Comment:
If the whole method was removed, shouldn't we handle the places that invoked this method?

Reply:
Answered: no need.